src/core/models.js

```diff
@@ -0,0 +1,33 @@
+import seer from 'seer';
+
+const models = {};
+
+export const addModel = model => {
+  if (models[model.id]) {
+    return;
+  }
+  models[model.id] = model;
+};
+
+export const removeModel = id => {
+  delete models[id];
+};
+
+const recursiveSet = (obj, path, value) => {
+  if (path.length > 1) {
+    recursiveSet(obj[path[0]], path.slice(1), value);
+  } else {
+    obj[path[0]] = value;
+  }
+};
+
+seer.listenFor('luma.gl', payload => {
```

Comment:
Suggest being very generous with documentation here. Both header and inside method.